### ISSUE TITLE
[Question] Is there a way to isolate a test code block?

### ISSUE THREAD

**Issue by yongmin86k** [AkshaySG14/haystack-public#22]
I love the editor and it really helps my coding experience.

One question about isolating the code block feature is when writing some tests.

Currently, isolating the test code scrolls to the point very far bottom. (could be a bug) 

eg)
env: nodejs, mocha, 
```javascript
describe('some tests', function() {
  it('dummy test1', function() {
    // ...
  })

  it('can isolate the test', function() {
    //
    // I would like to isolate this test block only but it is not working.
    //
  })

  it('dummy test2', function() {
    // ...
  })
})
```

**Comment by AkshaySG14** [comment]
This looks like a bug; will fix.

**Comment by AkshaySG14** [comment]
This should be fixed in the latest version on the website: 0.17! Please let me know if it is not.
